sessions/backend-update-delete/demo-end/components/ListItem/index.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const StyledListItem = styled.li`
+  width: 100%;
```

Comment:
I am not sure but do we need the with 100% here? i thought list item are block elements.

Reply:
I'll test it!

Reply:
So, this is behavior specific to `<li>` apparetly, because it's neither `display: block`, nor `display: inline-block`, it's `display: list-item`, which apparently shares some default behavior with inline-block elements. 🤷‍♀️ Wild.